src/types/onyx/Task.ts

```diff
@@ -2,7 +2,7 @@ import type Report from './Report';
 
 type Task = {
     /** Title of the Task */
-    title: string;
+    title?: string;
```

Comment:
Why was this changed?

Reply:
We can change the description without the title